kstreams-live-update/aggregator/src/main/java/io/debezium/examples/kstreams/liveupdate/aggregator/StreamsPipeline.java

```diff
@@ -67,4 +77,35 @@
                         .divide(BigDecimal.valueOf(100), 2, RoundingMode.HALF_UP))
                 .mapValues(v -> String.valueOf(v));
     }
+
```

Comment:
Do we really need to check the presence of the topics?

Reply:
Yes, I think so. The KStreams pipeline fails otherwise. That's why there was the shell step in it before.